src/components/Card.js

```diff
@@ -16,14 +25,17 @@ export default class Card {
     this._cardElement = null;
   }
 
```

Comment:
Note, that now you should update likes on the server.

The code for rendering likes should look like this:

```js
// in Card class 

// call this method from index.js after successful api response in `then` block
updateLikes(likes) { 
   // set instance variable
    this._likes = likes; 
    this._renderLikes(); 
} 

isLiked() {
 // return true if user liked the card, otherwise false
}

_renderLikes() {
  //set  likes counter content using this._likes.length

  if (this.isLiked()) {
    // add active class to like button
  } else {
    // remove active class from like button
  }
}

generateCard() { 
  ...
  // render initial likes, you saved them in the constructor
  this._renderLikes()
}

// index.js
if (card.isLiked()) {
  api.changeCardLike(card._id, "DELETE").then(response => card.updateLikes(response.likes)).catch(() => /*handle error*/)
} else {
 // the same for adding like
}
```

Reply:
Kseniya, 

Thank you so much for the thorough reviews - it was so helpful to go over them! It seems like they updated the server for sprint 09 and there is no longer "card likes" on card data. I only see "isLiked" so I made it update based on the click to be true or false. 

Not entirely sure if what I have is correct, so I would appreciate it if you have more notes! 

Thanks again! 
Alma

Reply:
Ok, I got it, thank you!